.editorconfig

```diff
@@ -11,6 +11,12 @@ root = true
 indent_style = space
 indent_size = 4
 
+# Default severity for analyzer diagnostics with category 'DocumentationAnalyzers.Refactorings'
+dotnet_analyzer_diagnostic.category-DocumentationAnalyzers.Refactorings.severity = suggestion
+dotnet_diagnostic.DOC100.severity = none # DOC100: Place text in paragraphs
+dotnet_diagnostic.DOC900.severity = none # DOC900: Render as Markdown (not working correctly at time of initial use)
```

Comment:
💭 Well it sure won't work like this. Since the default severity is `refactoring`, maybe leave this one at its default?